### PR TITLE
fix(); aave forks; fix linting issues

### PR DESCRIPTION
**Pull request by dmelotik**

### CODE REVIEW

**Review by dmelotik — COMMENTED** [review]
@steegecs Interesting thing to note here: In a handler you NEED to include the event and proper type even if you do not use the event. 

Is there any way to skip this in the linter?

**Comment by jaimehgb** [comment]
> @steegecs Interesting thing to note here: In a handler you NEED to include the event and proper type even if you do not use the event.
> 
> Is there any way to skip this in the linter?

@dmelotik You can explicitly tell the linter to ignore a line (or a file). Add this comment on the line right above:
```
// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
(or just `eslint-disable-next-line`). This is one scenario when it can be justified to ignore the linter :D

**Comment by dmelotik** [comment]
yay breaking shit, thanks @jaimehgb

File: subgraphs/aave-v2-forks/protocols/aave-v2/src/mapping.ts [hunk 1]
```diff
@@ -79,15 +79,15 @@ import { IPriceOracleGetter } from "../../../generated/LendingPool/IPriceOracleG
 import { Transfer } from "../../../generated/templates/AToken/AToken";
 
 function getProtocolData(): ProtocolData {
-  let letants = getNetworkSpecificConstant();
```
Comment (jaimehgb) [inline subgraphs/aave-v2-forks/protocols/aave-v2/src/mapping.ts:82]: lol

**Review by jaimehgb — APPROVED** [review]

File: subgraphs/aave-v2-forks/protocols/aave-v2/src/mapping.ts (hunk 1 above)
Comment (dmelotik) [reply subgraphs/aave-v2-forks/protocols/aave-v2/src/mapping.ts:82]: I think that happened when i was doing a ctrl+f change lmao

**Pull request merged**